tests/Maker/FunctionalTest.php

```diff
@@ -387,6 +395,94 @@ function (string $output, string $directory) {
                 ),
         ];
 
+        $makerTestAuthenticatorLoginFormUserEntity = MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeAuthenticator::class),
+            [
+                // authenticator type => login-form
+                1,
+                // class name
+                'AppCustomAuthenticator',
+                // controller name
+                'SecurityController',
+            ]
+        )
+            ->addExtraDependencies('doctrine')
+            ->addExtraDependencies('security')
+            ->addExtraDependencies('twig')
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeAuthenticatorLoginFormUserEntity')
+            ->configureDatabase()
+            ->updateSchemaAfterCommand()
+            ->assert(
+                function (string $output, string $directory) {
+                    $this->assertContains('Success', $output);
+
+                    $fs = new Filesystem();
+                    $this->assertTrue($fs->exists(sprintf('%s/src/Controller/SecurityController.php', $directory)));
+                    $this->assertTrue($fs->exists(sprintf('%s/templates/security/login.html.twig', $directory)));
+                    $this->assertTrue($fs->exists(sprintf('%s/src/Security/AppCustomAuthenticator.php', $directory)));
+                }
+            );
+
+        if (Kernel::VERSION_ID < 40100) {
+            $makerTestAuthenticatorLoginFormUserEntity->addExtraDependencies('symfony/form');
+        }
+
+        yield 'auth_login_form_user_entity' => [
+            $makerTestAuthenticatorLoginFormUserEntity,
+        ];
+
+        yield 'auth_login_form_user_not_entity' => [
+            MakerTestDetails::createTest(
+                $this->getMakerInstance(MakeAuthenticator::class),
+                [
+                    // authenticator type => login-form
+                    1,
+                    // class name
+                    'AppCustomAuthenticator',
+                    // controller name
+                    'SecurityController',
+                    // user class
+                    'App\Security\User'
+                ]
+            )
+                ->addExtraDependencies('security')
+                ->addExtraDependencies('twig')
+                ->addExtraDependencies('doctrine/annotations')
+                ->addExtraDependencies('symfony/form')
+                ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeAuthenticatorLoginFormUserNotEntity')
+                ->assert(
+                    function (string $output) {
+                        $this->assertContains('Success', $output);
+                    }
```

Comment:
I don't think this is necessary. I think we either already check for this always, or, we are at least checking that the process exited successfully, which is basically the same as this.